lightning/src/util/persist.rs

```diff
@@ -0,0 +1,81 @@
+// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
+// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// You may not use this file except in accordance with one or both of these
+// licenses.
+
+//! Traits for handling persistence
```

Comment:
Can we expand on this somewhat? Lets say something about how we expose a simple `KVStore` trait which can be used to implement both a graph + manager persistence and `chain::Persist` all in one simple implementation.